resources/web/lib/prettify/prettify.js

```diff
@@ -1731,6 +1731,93 @@ var prettyPrint;
              : (prettyPrint = $prettyPrint)
       };
 
+// BEGIN TABBED WIDGET JS
```

Comment:
This file is all third-party code from Google that we've copied into the repo. I believe the best way to add this is to add a new file under `resources/web/docs_js/components/` and include an import at the top of `resources/web/docs_js/index.js`.

Sorry for the runaround. I meant to look at the right place to do this before you made any changes, but you're too fast! 🏃🏻

Reply:
It should end up in the same place in the output: `static/docs.js`.

Reply:
Awesome! Thank you. No worries, that's my fault for trying to figure it out on my own.

Reply:
The code in `docs_js/components` seems to be for functionality that is actually coded into the doc build -- like the console widgets. Are you sure this is where this JS should go?

Reply:
To clarify, it does work – I just want to call out the difference.

Reply:
Moved in d8b9683.

Reply:
> The code in `docs_js/components` seems to be for functionality that is actually coded into the doc build, like the console widgets. Are you sure this is where this JS should go?

I'm not sure, but I think so. Aren't both this and the console widgets pieces of interactive content in the docs? I guess I'm not sure what you see as the difference.

Reply:
Maybe this doesn't matter, but the difference I see is that the console widget is something that the doc build converts from Asciidoc to HTML. On the other hand, the tabbed widget is raw HTML that we add to an Asciidoc file; the doc build doesn't need to converting anything.